Assignment2Q1.java

```diff
@@ -0,0 +1,117 @@
+import java.util.LinkedList;
+import java.util.List;
+
+public class Assignment2Q1 {
+	public static void main(String[] args) {
+		BinaryTreeNode<Integer> n1 = new BinaryTreeNode<>(1, null, null);

```

Comment:
I mean, that you can avoid writing _null_'s there. Cuz with your 1-arg. costructor it may be the same.